GitCommands/Settings/ConfigFileSettings.cs

```diff
@@ -5,51 +5,53 @@
 
 namespace GitCommands.Settings
 {
+    [DebuggerDisplay("{" + nameof(SettingLevel) + "} {" + nameof(SettingsCache) + "} << {" + nameof(LowerPriority) + "}")]
```

Comment:
Do you think you could add a screenshot of how it looks?

Reply:
![image](https://github.com/gitextensions/gitextensions/assets/36601201/dc9281d8-a46d-4ce1-baa2-303683e134cc)

Reply:
What would this look like?
```suggestion
    [DebuggerDisplay("{" + nameof(SettingLevel) + "} ({" + nameof(SettingsCache) + "}) <- {" + nameof(LowerPriority) + "}")]
```

Reply:
> What would this look like?

Only if you really insist:

![image](https://github.com/gitextensions/gitextensions/assets/36601201/3dfcb076-bb16-4226-bf28-f7b04328e95e)

or:

![image](https://github.com/gitextensions/gitextensions/assets/36601201/2635692a-83f9-4e6e-9c17-c7cfef2dbd15)

Reply:
Igor, do you accept
![image](https://github.com/gitextensions/gitextensions/assets/36601201/5e32bac8-da11-45f1-a5a3-5df01c095af0)
?